src/hash.h

```diff
@@ -213,12 +213,12 @@ class CHashWriterYescrypt: public CHashWriter
     uint256 GetHash() {
         uint256 result;
         assert(buf.size() == 80);
```

Comment:
This is an unnecessary line.
I heard that this line was for debugging, which was added by old BitZeny developers.

Reply:
Thank you for your review! :+1: 

I still do not understand why this should be removed. Could you explain it in more detail?

Is your request is remove just this one line? or the whole function? Let me know exactly what code to remove and I will test it.

Reply:
Could you tell me who added this? And I want to see the commit log.

Reply:
The original `bitcoin 0.15` code was here. you may right.

```c
    uint256 GetHash() {
        uint256 result;
        ctx.Finalize((unsigned char*)&result);
        return result;
    }
```
https://github.com/bitcoin/bitcoin/blob/b6677651515534ed81b182467b62a5fd3f42d4b9/src/hash.h#L149

Reply:
i asked to @fish-en

Reply:
@shuichiro-makigaki Could you fix this file for us? I am no expert in this file...

Reply:
@shuichiro-makigaki Thank you for your explanation. I will remove this line and test again.

Reply:
it seems ok. i will update soon